src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ApplicationsController.kt

```diff
@@ -243,10 +243,16 @@ class ApplicationsController(
     return ResponseEntity.ok(assessmentTransformer.transformJpaToApi(assessment, offender, inmate))
   }
 
-  private fun getPersonDetail(crn: String): Pair<OffenderDetailSummary, InmateDetail> {
+  private fun getPersonDetail(crn: String, ignoreLaoQualification: Boolean = false): Pair<OffenderDetailSummary, InmateDetail> {
     val user = userService.getUserForRequest()
 
-    val offenderResult = offenderService.getOffenderByCrn(crn, user.deliusUsername, user.hasQualification(UserQualification.LAO))
+    val ignoreLao = if (ignoreLaoQualification) {
+      false
+    } else {
+      user.hasQualification(UserQualification.LAO)
+    }
```

Comment:
I found it a bit difficult to understand what the logic was here. I think renaming `ignoreLaoQualification` to something like `forceLaoChecks` would make it clearer what the intent is.